src/generators/breadcrumbs-generator.php

```diff
@@ -165,6 +174,107 @@ public function generate( Meta_Tags_Context $context ) {
 		}, $crumbs, array_keys( $crumbs ) );
 	}
 
+	/**
+	 * Returns the modified post crumb.
+	 *
+	 * @param array     $crumb    The crumb.
+	 * @param Indexable $ancestor The indexable.
+	 *
+	 * @return array The crumb.
+	 */
+	private function get_post_crumb( $crumb, $ancestor ) {
+		$crumb['id'] = $ancestor->object_id;
+
+		return $crumb;
+	}
+
+	/**
+	 * Returns the modified post type crumb.
+	 *
+	 * @param array     $crumb    The crumb.
+	 * @param Indexable $ancestor The indexable.
+	 *
+	 * @return array The crumb.
+	 */
+	private function get_post_type_archive_crumb( $crumb, $ancestor ) {
+		$crumb['ptarchive'] = $ancestor->object_sub_type;
+
+		return $crumb;
+	}
+
+	/**
+	 * Returns the modified term crumb.
+	 *
+	 * @param array     $crumb    The crumb.
+	 * @param Indexable $ancestor The indexable.
+	 *
+	 * @return array The crumb.
+	 */
+	private function get_term_crumb( $crumb, $ancestor ) {
+		$crumb['term_id'] = $ancestor->object_id;
+
+		return $crumb;
+	}
+
+	/**
+	 * Returns the modified system page crumb.
+	 *
+	 * @param array     $crumb    The crumb.
+	 * @param Indexable $ancestor The indexable.
+	 *
+	 * @return array The crumb.
+	 */
+	private function get_system_page_crumb( $crumb, $ancestor ) {
+		if ( $ancestor->object_sub_type === 'search-result' ) {
+			$crumb['text'] = $this->options->get( 'breadcrumbs-searchprefix' ) . ' “' . \esc_html( \get_search_query() ) . '”';
+			$crumb['url']  = \get_search_link();
+		}
+		elseif ( $ancestor->object_sub_type === '404' ) {
+			$crumb['text'] = $this->options->get( 'breadcrumbs-404crumb' );
+		}
+
+		return $crumb;
+	}
+
+	/**
+	 * Returns the modified user crumb.
+	 *
+	 * @param array     $crumb    The crumb.
+	 * @param Indexable $ancestor The indexable.
+	 *
+	 * @return array The crumb.
+	 */
+	private function get_user_crumb( $crumb, $ancestor ) {
+		$display_name = \get_the_author_meta( 'display_name', $ancestor->object_id );
+		$crumb['text'] = $this->options->get( 'breadcrumbs-archiveprefix' ) . ' “' . $display_name . '”';
```

Comment:
During testing I found that the `“` quotes were not present in earlier versions of our plugin ( 13.5 specifically ).
```suggestion
		$crumb['text'] = $this->options->get( 'breadcrumbs-archiveprefix' ) . ' ' . $display_name;
```